pom.xml

```diff
@@ -0,0 +1,158 @@
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>${project.groupId}:${project.artifactId}</name>
+  <description>Support for integrating OpenTracing in Web Servlet projects</description>
+  <url>http://github.com/opentracing-contrib/java-web-servlet-filter</url>
+
+  <scm>
+    <url>https://github.com/opentracing-contrib/java-web-servlet-filter</url>
+    <connection>scm:git:https://github.com/opentracing-contrib/java-web-servlet-filter.git</connection>
+    <developerConnection>scm:git:https://github.com/opentracing-contrib/java-web-servlet-filter.git</developerConnection>
+    <tag>HEAD</tag>
+  </scm>
+
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <id>pavolloffay</id>
+      <name>Pavol Loffay</name>
+      <email>p.loffay@gmail.com</email>
+    </developer>
+  </developers>
+
+  <properties>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <version.io.opentracing>0.20.7</version.io.opentracing>
+    <version.javax.servlet-javax.servlet-api>3.0.1</version.javax.servlet-javax.servlet-api>
+    <version.junit>4.12</version.junit>
+
+    <version.maven-deploy-plugin>2.8.2</version.maven-deploy-plugin>
+    <version.maven-release-plugin>2.5.3</version.maven-release-plugin>
+    <version.maven-source-plugin>3.0.1</version.maven-source-plugin>
+    <version.maven-javadoc-plugin>2.10.4</version.maven-javadoc-plugin>
+    <version.io.zikin.centralsync-maven-plugin>0.1.0</version.io.zikin.centralsync-maven-plugin>
+  </properties>
+
+  <modules>
+    <module>opentracing-web-servlet-filter</module>
+  </modules>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>${project.groupId}</groupId>
+        <artifactId>opentracing-web-servlet-filter</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>io.opentracing</groupId>
+        <artifactId>opentracing-api</artifactId>
+        <version>${version.io.opentracing}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.opentracing</groupId>
+        <artifactId>opentracing-mock</artifactId>
+        <version>${version.io.opentracing}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>junit</groupId>
+        <artifactId>junit</artifactId>
+        <version>${version.junit}</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <profiles>
+    <profile>
+      <id>release</id>
+
+      <distributionManagement>
+        <repository>
+          <id>bintray</id>
+          <url>https://api.bintray.com/maven/opentracing/maven/opentracing-java/;publish=1</url>
```

Comment:
Same comment as in https://github.com/opentracing-contrib/java-jaxrs/pull/5,
shouldn't `opentracing-java` be changed to `java-web-servlet-filter` in this case?